app/javascript/components/CaseNoteCardModal.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import MUIRichTextEditor from 'mui-rte';
+import 'draft-js/dist/Draft.css';
+import 'draftail/dist/draftail.css';
+import { Button, TextField, Dialog, DialogActions, DialogContent, DialogContentText, Grid, Paper } from '@material-ui/core/';
+import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles';
+
+const styles = {
+    buttonStyle: {
+        marginLeft: 'auto',
+        marginRight: '0',
+    },
+    casenoteCardStyle: {
+        marginLeft: '20px',
+        padding: '20px',
+        boxShadow: '0px 4px 10px rgba(0, 0, 0, 0.15)',
+        borderRadius: '10px',
+        height: '200px',
+    },
+    casenoteDescStyle: {
+        marginLeft: '20px',
+        paddingTop: '20px',
+    },
+    dialogActionsStyle: {
+        padding: '30px',
+    },
+    MUIRichTextEditorStyle: {
+        border: '5px solid',
+        padding: '10px',
```

Comment:
The styles related to MUIRichTextEditor show up in a lot of different files. Would be more manageable to wrap the MUIRichTextEditor in its own component or centralize it so if we decide to change some style we only have to change it in one place

Reply:
^ note goes for most repeated styles also. the textEditor was just the one i noticed